pkgs/by-name/op/openwebrxplus/package.nix

```diff
@@ -0,0 +1,204 @@
+        --replace ', "fftw3"' ""
+    '';
+
+    propagatedBuildInputs = [ pycsdr ];
+    buildInputs = [ csdr-eti csdr ];
+    NIX_CFLAGS_COMPILE = [ "-I${pycsdr}/include/${python.libPrefix}" ];
+
+    # has no tests
+    doCheck = false;
+    pythonImportsCheck = [ "csdreti" ];
+
+    meta = {
+      homepage = "https://github.com/jketterl/pycsdr";
+      description = "bindings for the csdr library";
+      license = lib.licenses.gpl3Only;
+      maintainers = lib.teams.c3d2.members;
+    };
+  };
+
+  pycsdr = buildPythonPackage rec {
+    pname = "pycsdr";
+    version = "0.18.23";
+    format = "setuptools";
+
+    src = fetchFromGitHub {
+      owner = "luarvique";
+      repo = "pycsdr";
+      rev = version;
+      hash = "sha256-NjRBC7bhq2bMlRI0Q8bcGcneD/HlAO6l/0As3/lk4e8=";
+    };
+
+    buildInputs = [ csdr ];
+
+    # has no tests
+    doCheck = false;
+    pythonImportsCheck = [ "pycsdr" ];
+
+    meta = {
+      homepage = "https://github.com/jketterl/pycsdr";
+      description = "bindings for the csdr library";
+      license = lib.licenses.gpl3Only;
+      maintainers = lib.teams.c3d2.members;
+    };
+  };
+
+  owrx_connector = stdenv.mkDerivation rec {
+    pname = "owrx_connector";
+    version = "0.6.5";
+
+    src = fetchFromGitHub {
+      owner = "luarvique";
+      repo = pname;
+      rev = version;
+      sha256 = "sha256-e0VEv9t4gVDxJEbDJm1aKSJeqlmhT/QimC3x4JJ6ke8=";
+    };
+
+    nativeBuildInputs = [ cmake ninja pkg-config ];
+
+    buildInputs =
+      [ libsamplerate fftwFloat csdr rtl-sdr soapysdr-with-plugins ];
+
+    meta = with lib; {
+      homepage = "https://github.com/jketterl/owrx_connector";
+      description =
+        "A set of connectors that are used by OpenWebRX to interface with SDR hardware";
+      license = licenses.gpl3Only;
+      platforms = platforms.unix;
+      maintainers = teams.c3d2.members;
+    };
+  };
+in buildPythonApplication rec {
+  pname = "openwebrxplus";
+  version = "1.2.70";
+
+  src = fetchFromGitHub {
+    owner = "luarvique";
+    repo = "openwebrx";
+    rev = version;
+    sha256 = "sha256-17Ug3xZ2tUMvwn3SUZqz8Y1Dd7UQrEsa5YQ7XV+FId0=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    dablin
+    direwolf
+    dump1090
+    hamlib
+    imagemagick
+    multimon-ng
+    nrsc5
+    rtl_433
+    wsjtx
+    setuptools
+    pycsdr
+    pycsdr-eti
+    pydigiham
+    js8py
+    owrx_connector
+    soapysdr-with-plugins
+  ];
+
+  buildInputs = [ direwolf sox wsjtx codecserver ];
+
+  pythonImportsCheck = [ "csdr" "owrx" "test" ];
+
+  passthru = { inherit js8py owrx_connector pycsdr csdr; };
+
+  meta = with lib; {
+    homepage = "https://github.com/luarvique/openwebrx";
+    description =
+      "A simple DSP library and command-line tool for Software Defined Radio";
+    mainProgram = "openwebrx";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ arcnmx kittywitch ] ++ teams.c3d2.members;
```

Comment:
I don't think we should copy the original maintainers over

Reply:
happy to help, tbh

Reply:
By the way, this was all taken from https://github.com/gensokyo-zone/infrastructure/blob/main/packages/openwebrxplus.nix https://github.com/gensokyo-zone/infrastructure/blob/main/modules/nixos/openwebrx.nix, which is why we are in as maintainers.

Reply:
That's right, thank you!

Sandro, me, et al. are the c3d2 team who already maintain the openwebrx package.